build/images/ovs/apply-patches.sh

```diff
@@ -80,8 +80,16 @@ if version_get "$OVS_VERSION" "2.13.0" && version_lt "$OVS_VERSION" "2.14.0" ; t
         git apply
 fi
 
+# Starting from version 5.7.0, strongSwan no more supports to specify a configuration parameter with
```

Comment:
s/strongSwan no more supports to specify a configuration/strongSwan no longer supports specifying a configuration

Reply:
Updated.